### DISCUSSION TITLE
JuliaQuantumControl Map for Beginners

### DISCUSSION THREAD

**Discussion by Fe-r-oz** [JuliaQuantumControl/QuantumControl.jl#69] [Show and tell]
Hi, @goerz 

In order to have a better sense about overall functionality for beginners, I started making a complete JuliaQuantumControl Map. 

Any suggestions are more than welcome! Only `storage` and `spectral rang`e set of methods from `Quantum Propagators `to be added are left with some singular methods.

Should we consider including a polished version of this in ReadMe? 

![QuantumControl9](https://github.com/JuliaQuantumControl/QuantumControl.jl/assets/93876775/60d999c2-5a2d-4e8c-aad6-4e00fff5c4b0)

**Comment by goerz** [comment]
I don't think that's something I would want to include in the documentation. A lot of those functions are internals. https://juliaquantumcontrol.github.io/QuantumControl.jl/stable/api/quantum_control/ and https://juliaquantumcontrol.github.io/QuantumPropagators.jl/stable/api/quantumpropagators/ should already provide this information with the intended structure.
